server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/Properties.scala

```diff
@@ -26,6 +26,8 @@ import play.api.libs.json.JsObject
 
 object Properties {
   def empty(): Properties = Properties(Set())
+
+  def of(values: NonEmptyString*): Properties = Properties(values.toSet)
```

Comment:
Could this be an `apply` method? So that I can even write `Properties("id", "size")`